README.md

```diff
@@ -386,6 +400,7 @@ The power of EB is that it will automatically update our environment to satisfy
 1. Tell EB to update your environment accordingly:
 
   ```
+  $ git add --all
   $ eb deploy --staged
```

Comment:
It works as long as you use `eb deploy --staged` which picks the files out of the staging area instead of the branch.  Paul pointed that out in issue #1 comments, and I verified it works.  It's a nice way to do experimental changes, but I can change it if you want...